daemon/Startup.cs

```diff
@@ -50,6 +50,7 @@
 using Spectero.daemon.Libraries.Core.Statistics;
```

Comment:
There's a merge conflict here, see my suggestion in issue body on how to resolve (somewhat easily).